did/cli.py

```diff
@@ -149,8 +149,15 @@ def parse(self):
             raise RuntimeError(
                 "Invalid date range ({0} to {1})".format(
                     opt.since, opt.until.date - delta(days=1)))
-        header = "Status report for {0} ({1} to {2}).".format(
+
+        header = "Status report for {0} ({1} to {2})".format(
             period, opt.since, opt.until.date - delta(days=1))
+        if opt.format == "md":
+            # in markdown first line must be an header
+            header = "# " + header
```

Comment:
I'd suggest to use the alternate syntax for headings as described here: https://www.markdownguide.org/basic-syntax/#alternate-syntax. Then you can use `did`'s output in commit messages as well. The `#` headers are always consumed as comments which is a bit unfortunate.

Reply:
done